src/components/general/ProgressBar.js

```diff
@@ -0,0 +1,24 @@
+import React from "react";
+import { makeStyles } from "@material-ui/core/styles";
+import LinearProgress from "@material-ui/core/LinearProgress";
+
+const useStyles = makeStyles(theme => ({
+  root: {
+    width: "100%",
+    "& > * + *": {
+      marginTop: theme.spacing(2)
+    },
+    textAlign: "center"
+  }
+}));
+
+export default function ProgressBar(props) {
+  const classes = useStyles();
+
+  return (
+    <div className={classes.root}>
+      {props.progressPercentage}%
+      <LinearProgress variant="determinate" value={props.progressPercentage} />
```

Comment:
Out of curiosity, what does `variant="determinate"` do? [The docs I looked at](https://material-ui.com/api/linear-progress/#linearprogress-api) weren't super clear.